backend-tests/tests/test_auditlogs.py

```diff
@@ -95,29 +87,77 @@ def test_deployment_create(self, tenant_users):
 
         check_log(res[0], expected)
 
+    def test_user_create(self, tenant_users):
+        user = tenant_users.users[0]
+
+        alogs = ApiClient(auditlogs.URL_MGMT)
+
+        uid = make_user(user.token, "foo@acme.com", "secretsecret")
+        expected = evt_user_create(user, uid, "foo@acme.com")
+
+        time.sleep(0.5)
+        res = alogs.with_auth(user.token).call("GET", auditlogs.URL_LOGS)
```

Comment:
... And here
```suggestion
        for _ in range(20):
            time.sleep(0.5)
            res = alogs.with_auth(user.token).call("GET", auditlogs.URL_LOGS)
            if res.status_code == 200:
                break
```

Reply:
very good suggestion, fixed all the related issues.

actually, I bit the bullet and pulled in a retry library - some cool utils for retries, which we all need in multiple places.

I chose mozilla/redo for the time being, liked it the most. it can handle both function retries(decorators) and loose code blocks (wrap them in a retrier generator loop, like I do).